server/src/main/java/org/tctalent/server/service/db/impl/DuolingoApiServiceImpl.java

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
Just one more comment relating to the Duolingo database definition. (Sorry I missed this when you first submitted it).

`CREATE TABLE duolingo_coupon (
                         id BIGSERIAL NOT NULL PRIMARY KEY,
                         coupon_code VARCHAR(255) UNIQUE NOT NULL,
                         candidate_id BIGINT,
                         expiration_date TIMESTAMP NOT NULL,
                         date_sent TIMESTAMP,
                         coupon_status VARCHAR(50) NOT NULL
                         );
`

If you take a look at our other Flyways you will see that we have a standard of using timestamptz instead of just timestamp. Whether or not that is a good idea, it is our current standard.

Also when using references to other tables - eg candidate_id - it is our standard to be explicit about the table being referenced:

eg
  candidate_id bigint references candidate

Reply:
Should I create a new migration for this? or just modifying it on migration 324?